Added some methods/cleanup

DanielGaull opened the pull request:
Added methods for mentioning users, mentioning the whole group, getting the group name, and getting the group members as a user ID list
Added storing of mentioned users, the user ID, and the group ID
Added some more comments

DanielGaull committed:
Added some methods/cleanup

Added methods for mentioning users, mentioning the whole group, getting the group name, and getting the group members as a user ID list
Added storing of mentioned users, the user ID, and the group ID
Added some more comments

william-reed commented on Base.gs:
```diff
@@ -1,29 +1,84 @@
+function mentionUser(message, username, userId) {
+  UrlFetchApp.fetch("https://api.groupme.com/v3/bots/post", {"method":"post", "payload":'{"bot_id":"' + botId + '","text": "@' + username + ' ' + message + '", ' + 
+                                                               '"attachments": [{"type": "mentions", "loci": [[0,' + 
+                                                             username.length + ']], "user_ids": ["' + userId + '"]}]}'});
```
i think some of this stuff might be a bit cleaner to read with a template literal instead of concatenating strings all over. what do you think?

DanielGaull replied:
Yeah I'll work on creating a quick example (probably an @all bot) and add it to the readme, then can clean up that long string

william-reed reviewed (commented):
Hey thanks for adding all this! For the most part it looks great. I left one comment but I'm not strict on it, let me know what you think. One other thing I think would be great would be to add an example utilizing all this in [here](https://github.com/william-reed/GroupMe-Bots-With-Google-Apps-Script/tree/master/Random%20Examples) and then add it to the list in `README.md` as well. That's totally up to you if your willing to do it or not. Let me know and we can get this merged!

DanielGaull committed:
Cleaned up mention post request

Now uses template literal for readability

DanielGaull committed:
Create MentionAll

Created an example that utilizes the mentioning functions, and will repeat any message a user sends that starts with "@everyone", pinging everyone in the group while doing so

DanielGaull committed:
Rename MentionAll to MentionAll.gs

DanielGaull committed:
Added an example

Added the @everyone example to the examples section

DanielGaull commented:
Should be up-to-date now

william-reed commented:
thanks for the contributions!
